### PR TITLE
add emoticon codepen and add colon functionality

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-435

Description of Changes:
* Add emoticon codepen to plugin page
* Add info on colon autocomplete functionality
* Convert double quotes to single quotes in code blocks

DOD:
- [x] nav.yml has been updated if applicable
- [x] file has been included where required if applicable
- [x] files removed have been deleted, not just excluded from the build if applicable

Review
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: plugins/emoticons.md [hunk 1]
```diff
@@ -9,20 +9,24 @@ controls: toolbar button
 
 This plugin adds a dialog to the editor lets users insert emojis into {{site.productname}}'s editable area. The dialog can be invoked via a toolbar button - `emoticons` - or a dedicated menu item added as `Insert > Emoticons`.
 
-> Note: it doesn't automatically convert text emoticons into graphical smilies.
+The emoticons plugin provides an autocompleter for adding emoticons without using the toolbar button or menu item. Adding a colon `:`, followed by at least two characters will open an inline dialog showing matching emoticons.
+
+> **Note**: The emoticons plugin does not automatically convert text emoticons into graphical emojis.
```
Comment (lnewson) [inline plugins/emoticons.md:14]: There's some inconsistency here with the use of `emoticons` and `emojis`.

File: plugins/emoticons.md [hunk 2]
```diff
@@ -9,20 +9,24 @@ controls: toolbar button
 
 This plugin adds a dialog to the editor lets users insert emojis into {{site.productname}}'s editable area. The dialog can be invoked via a toolbar button - `emoticons` - or a dedicated menu item added as `Insert > Emoticons`.
 
-> Note: it doesn't automatically convert text emoticons into graphical smilies.
+The emoticons plugin provides an autocompleter for adding emoticons without using the toolbar button or menu item. Adding a colon `:`, followed by at least two characters will open an inline dialog showing matching emoticons.
```
Comment (lnewson) [inline plugins/emoticons.md:12]: `inline dialog` is the wrong term here, it should be an `inline toolbar`, `popup toolbar` or something like that instead. I'm honestly not sure what the term is for this, as we've never really defined it.

Comment (tylerkelly13) [reply plugins/emoticons.md]: updated per our discussion to: emoticon picker

File: plugins/emoticons.md (hunk 1 above)
Comment (tylerkelly13) [reply plugins/emoticons.md:14]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1494/commits/4a7917b87cbf3216748ab8f1deba41caa1274003

File: plugins/emoticons.md [hunk 3]
```diff
@@ -9,20 +9,24 @@ controls: toolbar button
 
 This plugin adds a dialog to the editor lets users insert emojis into {{site.productname}}'s editable area. The dialog can be invoked via a toolbar button - `emoticons` - or a dedicated menu item added as `Insert > Emoticons`.
 
-> Note: it doesn't automatically convert text emoticons into graphical smilies.
+The emoticons plugin provides an autocompleter for adding emoji without using the toolbar button or menu item. Adding a colon `:`, followed by at least two characters will open the emoticon picker showing matching emoji.
```
Comment (ltrouton) [inline plugins/emoticons.md:12]: Should `emoji` be plural here?

Comment (tylerkelly13) [reply plugins/emoticons.md:12]: the plural of emoji can be either emoji or emojis. though line 10 conflicts at the moment...

Comment (ltrouton) [reply plugins/emoticons.md:12]: Yeah, that is what made me comment. I personally, don't have a preference as long as it consistent.

Comment (tylerkelly13) [reply plugins/emoticons.md:12]: s/emojis/emoji/g excluding codeblocks

**Pull request merged**